qkit/drivers/Tabor_WX1284C.py

```diff
@@ -985,7 +985,7 @@ def do_get_reference_source_freq(self):
             (string) : 10e6, 20e6, 50e6, 100e6
         '''
         logging.debug(__name__ + ' : Get clock reference frequency.')
-        return float(self.ask(':ROSC:FREQ ?'))
+        return int(self.ask(':ROSC:FREQ ?')[:-1])*1.0e6
```

Comment:
What is the tabor returning? I thought it's something like `10e6`and if you make `[:-1]`, it's only `10e`which will fail to be converted to int. `int(float(self.ask(':ROSC:FREQ ?')))` should do the job, or am I wrong?

Reply:
It returns a string like "10M" for 10e6 Hz for example. At least for the WX2184C model (maybe it is different for WX1284C?).